lib/tasks/load_tags_from_content_api.rake

```diff
@@ -0,0 +1,39 @@
+desc "Load tags from Content API into Collections Publisher."
+task :load_tags_from_content_api => :environment do
+  require 'gds_api/content_api'
+
+  content_api_endpoint = Plek.new.find("content_api")
+  content_api = GdsApi::ContentApi.new(content_api_endpoint)
+
+  puts "There are currently #{Tag.count} tags in the database"
+  puts "Fetching all tags from the Content API..."
+  tags = content_api.get_list!("#{content_api_endpoint}/tags.json")
```

Comment:
This will pull in all tags from content_api.  We probably need to filter these by the types we're interested in (section/subsection, and topic/subtopic).